QA.md

```diff
@@ -123,3 +123,268 @@
+            > You have received 0.09 HTR and 0.01 TTT on a new transaction.
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TTT - Test Token Test` name should be in gray
+    1. Click on the `TTT - Test Token Test` item
+        1. Nothing should happen
+    1. Click on the `HTR - HATHOR` item
+        1. The **Balance** page should open
+1. **Send 3 tokens after turn on `Show amounts on notification` option**
+    1. Turn on the `push-notification` feature toggle
+    1. Go to the **Push Notification** page
+    1. Turn **on** the `Enable Push Notification` option
+    1. Turn **on** the `Show amounts on notification` option
+    1. Send HTR, TTT and TN1 to this wallet
+        1. Wait 3-5 min
+        1. You should receive a notification of new transaction showing 2 amounts in the message
+            > You have received 0.05 TN1, 0.03 TTT and 1 other token on a new transaction.
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TTT - Test Token Test` name should be in gray
+        1. The `TN1 - Test Nft` name should be in gray
+    1. Click on the `TTT - Test Token Test` item
+        1. Nothing should happen
+    1. Click on the `TN1 - Test Nft` item
+        1. Nothing should happen
+    1. Click on the `HTR - HATHOR` item
+        1. The **Balance** page should open
+1. **Send 4 tokens after turn on `Show amounts on notification` option**
+    1. Turn on the `push-notification` feature toggle
+    1. Go to the **Push Notification** page
+    1. Turn **on** the `Enable Push Notification` option
+    1. Turn **on** the `Show amounts on notification` option
+    1. Send HTR, TTT, TN1 and TNT to this wallet
+        1. Wait 3-5 min
+        1. You should receive a notification of new transaction showing 2 amounts in the message
+            > You have received 0.08 TNT, 0.05 TN1 and 2 other tokens on a new transaction.
+    1. Click on the notification
+    1. Wait until the modal with tx details open
+        1. The `HTR - HATHOR` name should be in the primary color (purple)
+        1. The `TTT - Test Token Test` name should be in gray
+        1. The `TN1 - Test Nft` name should be in gray
+        1. The `TNT - Test Nft Test` name should be in gray
+    1. Click on the `TTT - Test Token Test` item
+        1. Nothing should happen
+    1. Click on the `TN1 - Test Nft` item
+        1. Nothing should happen
+    1. Click on the `TNT - Test Nft Test` item
+        1. Nothing should happen
+    1. Click on the `HTR - HATHOR` item
+        1. The **Balance** page should open
+1. **Register `TTT` token and send 2 tokens after turn on `Show amounts on notification` option**
+    > WARNING: Not possible using `wallet-service` due to a validation that consults the fullnode.
```

Comment:
Only in the testnet, right?

Reply:
Yes, only in testnet. I will add this.

Reply:
[6638db6](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/248/commits/6638db6d32dc3a51876cbd3c8b180181fd321dbc)